autoload/lsp.vim

```diff
@@ -298,15 +298,28 @@ function! s:ensure_init(buf, server_name, cb) abort
         return
     endif
 
+    if has_key(l:server_info, 'capabilities')
+        let l:capabilities = l:server_info['capabilities']
+    else
+        let l:capabilities = {}
+    endif
+
+    if has_key(l:server_info, 'initialization_options')
```

Comment:
can you give an example of the server that uses this?

Reply:
[cquery](https://github.com/jacobdufault/cquery) uses at least `initializationOptions`, but both are part of the langugae server protocol for [initialize](https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#initialize).

Reply:
Great. Could you add a wiki page on how to use cquery at https://github.com/prabirshrestha/vim-lsp/wiki/Servers